core/src/miner/mem_pool.rs

```diff
@@ -269,7 +268,10 @@ impl MemPool {
 
             let id = self.next_transaction_id;
             self.next_transaction_id += 1;
-            let item = MemPoolItem::new(tx, origin, inserted_block_number, inserted_timestamp, id, timelock);
+            let item = MemPoolItem::new(tx, origin, inserted_block_number, inserted_timestamp, id, TxTimelock {
```

Comment:
I think rebasing this PR on https://github.com/CodeChain-io/foundry/pull/11 will make this commit clear.

Reply:
I'll try it when #11 is merged.